sdk/resourcemanager/gulpfile.js

```diff
@@ -122,11 +122,15 @@ function codegen(project, cb) {
 
     const regenManager = args['regenerate-manager'] ? ' --regenerate-manager ' : '';
 
+    // use following option in api-specs.json, if modelerfour reports PreCheck error (current seems occur to containerinstance, sql).
+    //--pipeline.modelerfour.additional-checks=false
+    //--pipeline.modelerfour.lenient-model-deduplication=true
```

Comment:
I've disabled the override in autorest.java

We might likely see m4 check failure on next time generation. We can decide later whether to put it here.